ReText/__init__.py

```diff
@@ -150,6 +162,44 @@ def getEditorFont(self):
 			font.fromString(self.editorFont)
 		return font
 
+class ReTextCache:
```

Comment:
Instead of introducing another class, what do you think about making a generic class and passing `settings` and `options` to its constructor?

Reply:
Yes, I think this is the logical thing to do!